File: app/src/main/java/com/example/pam_project/comparators/ListInformationComparator.java
package com.example.pam_project.comparators;

import com.example.pam_project.features.lists.list.ListInformation;

import java.util.Comparator;

public abstract class ListInformationComparator implements Comparator<ListInformation> {

    public int nullCompare(Object o1, Object o2) {
        if (o2 == null)
            return Integer.MIN_VALUE;
        if (o1 == null)
            return Integer.MAX_VALUE;
        return 0;
    }
}

File: app/src/main/java/com/example/pam_project/comparators/ListInformationComparator.kt
package com.example.pam_project.comparators

import com.example.pam_project.features.lists.list.ListInformation
import java.util.*

abstract class ListInformationComparator : Comparator<ListInformation?> {
    fun nullCompare(o1: Any?, o2: Any?): Int {
        if (o2 == null) return Int.MIN_VALUE
        return if (o1 == null) Int.MAX_VALUE else 0
    }
}